README.md

```diff
@@ -58,7 +58,7 @@ rho0 = qt.coherent_dm(n, alpha0)
 tsave = np.linspace(0, 1.0, 101)
 
 # uncomment the next line to run the simulation on GPU
-# torch.set_default_device('gpu')
+# torch.set_default_device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
Maybe we can keep the line uncommented with @abocquet proposed fix, and change the comment
```python
# run the simulation on a GPU if available, otherwise use the CPU
torch.set_default_device('cuda' if torch.cuda.is_available() else 'cpu')
```